docs/examples/and.rst

```diff
@@ -42,18 +42,18 @@ formulation of a BQM for a Boolean gate in detail. This example briefly repeats
 of mathematically formulating a BQM while adding details on the underlying physical processes.
 
 A D-Wave quantum processing unit (:term:`QPU`) is a chip with interconnected qubits; for example,
-a D-Wave 2000Q has up to 2048 qubits connected in a :term:`Chimera` topology. Programming it
+an Advantage QPU has over 5000 qubits connected in a :term:`Pegasus` topology. Programming it
 consists mostly of setting two inputs:
 
 * Qubit bias weights: control the degree to which a qubit tends to a particular state.
 * Qubit coupling strengths: control the degree to which two qubits tend to the same state.
 
 The biases and couplings define an energy landscape, and the D-Wave quantum computer seeks
-the minimum energy of that landscape. Once you express your problem in a formulation\ [#]_
+the minimum energy of that landscape. Once you express your problem in a formulation\ [1]_
```

Comment:
```suggestion
the minimum energy of that landscape. Once you express your problem in a formulation\ [#ising]_
```
(the second half of the comment below; GH doesn't allow suggested change over two diff snippets)